plugins/modules/gitlab_project.py

```diff
@@ -44,6 +44,11 @@
       - This option is only used on creation, not for updates.
     type: path
     version_added: "4.2.0"
+  build_timeout:
+    description:
+      - Maximum number of seconds a CI job can run.
```

Comment:
Something like:
```suggestion
      - Maximum number of seconds a CI job can run.
      - If not specified there will be no limit on how long the CI job can run.
```
could be added. It is kinda obvious, I know, but it might help newcomers.

Reply:
That's not correct I think; if not specified the current value will not be modified (on update), resp. GitLab's default value will be used (on creation). That might be no limit, but it might also be a specific limit.

Reply:
Ok, so I stand corrected. In any case, the description should say what the behaviour is when not specified.

Reply:
I've just added an extra documentation note, let me know how the wording sounds.

Reply:
Thanks!